extensions/google-search/src/index.tsx

```diff
@@ -38,6 +31,7 @@ export default function Command() {
                   />
 
                   <CopyToClipboardAction title="Copy URL to Clipboard" content={item.url} />
+                  <CopyToClipboardAction title="Copy query to Clipboard" content={item.query} />
```

Comment:
Shouldn't it be the suggestion and not the query?

Reply:
Good spot! That's updated